library/templates/dcp_mappluto.yml

```diff
@@ -2,47 +2,36 @@ dataset:
   name: &name dcp_mappluto
   version: "{{ version }}"
   acl: public-read
-  # Source definition
   source:
     url:
-      # The url or local path for this data source
       path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyc_mappluto_{{ version }}_unclipped_shp.zip
```

Comment:
This link is invalid 
<img width="1466" alt="Screen Shot 2022-09-07 at 10 46 18 AM" src="https://user-images.githubusercontent.com/41444592/188908052-e8c709fa-0c98-4056-bdca-5034beb9a7b2.png">